### ISSUE TITLE
Spider: Illinois International Port District

### ISSUE THREAD

**Issue by pjsier** [City-Bureau/city-scrapers#675]
URLs:

- Minutes: https://www.iipd.com/about/board-meeting-minutes
- Agendas: https://www.iipd.com/calendar/agendas
- Schedule: https://www.iipd.com/calendar/schedules

Spider Name: `il_port_district`
Agency Name: Illinois International Port District

See the [contribution guide](https://cityscrapers.org/docs/development/#contribute) for information on how to get started

**Comment by paqman85** [comment]
I'll take this one if it's still available.

**Comment by pjsier** [comment]
@paqman85 all yours!

**Comment by pjsier** [comment]
@paqman85 are you still working on this? Otherwise I'll take a stab at it

**Comment by paqman85** [comment]
Yes, im still on it. A few hiccups but I'm making progress now.

:)

**Comment by radoslawkrolikowski** [comment]
This issue is unclaimed and seems that no one is active. I would like to have a go at this, if it is available?

**Comment by pjsier** [comment]
@radoslawkrolikowski yep, it's been inactive long enough that it's all yours. Assigning you now

**Comment by radoslawkrolikowski** [comment]
I encountered problems while calling `pytest`. The following is the one of the errors that regards the `legistar`:
![image](https://user-images.githubusercontent.com/55454254/69454810-5cec3d00-0d67-11ea-93f8-e0241ecba35c.png)
My local fork is up-to-date with the upstream, I also installed requirements one more time in my virtual env, in case something has changed. I would like to emphasize that I call `pytest` on the repository that is identical to the upstream, I didn't make any changes. Do you have an idea how to deal with this issue?

I have also other question, for the sake of clarification: if I my `start_urls` list consist of three elements, then I should supply three different HTML files to the `tests/files` directory? And how to include this in the test file? 

Have you got any example that shows how to change following lines, so that it will consecutively call all my start urls?
```
test_response = file_response(
    join(dirname(__file__), "files", "chi_boardofethics.html"),
    url='https://www.chicago.gov/city/en/depts/ethics/supp_info/minutes.html'
) 
```
While my parse function looks like this:
```
    def parse(self, response):
        if response.url == self.start_urls[0]:
            return self.parse_agendas(response)
        if response.url == self.start_urls[1]:
            return self.parse_minutes(response)
        if response.url == self.start_urls[2]:
            return self.parse_schedules(response)
```

**Comment by pjsier** [comment]
You might need to re run pipenv sync because city-scrapers-core was updated.

For start URLs, we would want all of the HTML files to be in the tests directory, and you might try looking at start_requests in the scrapy documentation as a way of doing something similar. If you need the results of each page to go to the next page this approach might not work though because the requests can be concurrent

**Comment by pjsier** [comment]
Closed by #931, thanks @radoslawkrolikowski!
